src/Pretty/Types.hs

```diff
@@ -159,13 +162,13 @@ instance PrettyAnn CST.Typ where
     braces' commaSym (prettyAnn <$> xtors)
   -- Refinement types
   prettyAnn (CST.TyXRefined _ CST.Data   tn Nothing xtors) =
```

Comment:
These additional `parens` around structural and refinement types should be unnecessary.